win32/devdoc/interlocked_hl_requirements.md

```diff
@@ -20,10 +20,10 @@ typedef bool (*INTERLOCKED_COMPARE_EXCHANGE_64_IF)(int64_t target, int64_t excha
 MOCKABLE_FUNCTION_WITH_RETURNS(, INTERLOCKED_HL_RESULT, InterlockedHL_Add64WithCeiling, int64_t volatile*, Addend, int64_t, Ceiling, int64_t, Value, int64_t*, originalAddend)(INTERLOCKED_HL_OK, INTERLOCKED_HL_ERROR);
```

Comment:
>volatile [](start = 96, length = 8)

volatile_atomic here and everywhere

Reply:
Fixed.

---
In reply to: [499933068](https://github.com/Azure/azure-c-pal/pull/39#discussion_r499933068) [](ancestors = 499933068)